src/entities/drawer/ui/DrawingBoard.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useRef, useMemo, FC } from 'react';
+import React, { FC, useState, useRef, useMemo } from 'react';
 import { StyleSheet, View } from 'react-native';
 
```

Comment:
The lines still blinking, AB for Pads, see the screen where's a lot of points and try to move making very long line